scripts/tasks/eslint.ts

```diff
@@ -5,8 +5,18 @@ import * as fs from 'fs';
 
 const files = [path.join(process.cwd(), constants.directory)];
 const storiesPath = path.join(process.cwd(), 'stories');
+const hasStorySubfolder =
+  fs.existsSync(storiesPath) &&
+  fs
+    .readdirSync(storiesPath)
+    .map(subfolder => {
+      const storyFiles = fs.readdirSync(path.join(storiesPath, subfolder));
+      return storyFiles.map(file => path.join(storiesPath, file));
+    })
+    .flat()
+    .some(file => constants.extensions.includes(path.extname(file)));
```

Comment:
Added this extra check which makes sure a valid file exists within the `stories/` subfolder because linter errors out when running for this package after the migration since the `stories/` subfolder won't contain any of the valid extensions (.ts,.tsx,.js,.jsx) and only contains one story with a `.mdx` extension. `just-scripts` `eslintTask` doesn't seem to support the `no-error-on-unmatched-pattern` eslint CLI argument so this is a workaround.

![image](https://user-images.githubusercontent.com/8649804/204640087-8ab5fd28-44bb-49e0-a39b-8fb93452c57b.png)